less/quicksilver/packages/components/mirror.less

```diff
@@ -40,11 +40,7 @@
 }
 
 .mirror-members-h2{
-  /*
-  We need to migrate downloads.git to webpack to allow us to do this
-  background-image:url(../../../images/eclipse_org/downloads/mirror-h2-bg.jpg);
-  */
-  background-image:url(../images/mirror-h2-bg.jpg);
+  background-image:url(/downloads/images/eclipse_org/downloads/mirror-h2-bg.jpg);
```

Comment:
Have you verified the output of this? When you have URLs in LESS, it typically reads from the images folder to include the asset. This doesn't look like it lines up to anything in the assets folder. This would mean on any sites that attempt to use this style, the image may not be included in the build.

Reply:
Yes this is working in the downloads repo. I guess it is working since the image is already there. I tried a bunch of combination of urls and this one worked. I can see that this would be a problem, I'll try to find something else.